src/sage/graphs/graph_plot.py

```diff
@@ -812,12 +858,20 @@ def set_edges(self, **edge_options):
                     # Now add all the loops at this vertex, varying their size
                     for lab, col, _ in local_labels:
                         x, y = self._pos[a][0], self._pos[a][1] - loop_size
-                        c = circle((x, y), loop_size, rgbcolor=col, **eoptions)
+
+                        estyle = self._options['edge_style']
+                        ethickness = self._options['edge_thickness']
+                        if style_key_edges is not None and ((style_key_edges and (x, y) in self._options['edge_styles']) or (not style_key_edges and lab in self._options['edge_styles'])):
```

Comment:
please improve the alignment.

Reply:
What do you mean by the alignment? Do you mean for the `if` statement? Should I break it down so it's something like:
```
if (style_key_edges is not None
    and ((style_key_edges and (x, y) in self._options['edge_styles'])
        or (not style_key_edges and lab in self._options['edge_styles']))):
```

Or are you thinking something else?

Reply:
yes, this is much easier to read this way.